<launch>
    <param name="package_path" value="$(find livox_hikcamera_cal)"/>
    <rosparam file="$(find livox_hikcamera_cal)/cfg/setup_caliboard.yaml" command="load"/>
    <rosparam file="$(find livox_hikcamera_cal)/cfg/setup_ros.yaml" command="load"/>

    <param name="/shared_parameter/center_x" value="0" />
    <param name="/shared_parameter/center_y" value="0" />
    <param name="/shared_parameter/center_z" value="0" />
    <param name="/shared_parameter/length_x" value="0" />
    <param name="/shared_parameter/length_y" value="0" />
    <param name="/shared_parameter/length_z" value="0" />
    <param name="/shared_parameter/rotate_x" value="0" />
    <param name="/shared_parameter/rotate_y" value="0" />
    <param name="/shared_parameter/rotate_z" value="0" />

    <node pkg="livox_hikcamera_cal" type="camera_color_fusion_point_cloud" name="camera_color_fusion_point_cloud" output="screen"/>
    
</launch>